### ISSUE TITLE
1.3.1 / F87 for input fields with mandatory visual indication via pseudo-element

### ISSUE THREAD

**Issue by JOP-Temesis** [w3c/wcag#3781]
Does the F87 fail test for the CSS override of a pseudo-element concern the case of an input field with aria-required attributes?  

What are your thoughts on this? 

/J

**Comment by mbgower** [comment]
@JOP-Temesis I believe we need some more context to properly answer your question. Can you please provide more details or an example?

In the meantime, please examine https://github.com/w3c/wcag/issues/3748 to consider whether the discussion there on the use of pseudo elements is relevant.

**Comment by JOP-Temesis** [comment]
I was thinking of this kind of example  
where the mandatory field indication is in a content css and the input field with aria-required="true".

```
<label for="xx">my label <span>::before</span></label> #CSS content : "*";
<input type="text" id="xx" aria-required="true">

``` 

I don't see in which case the styles are replaced by users who could knowingly delete the *. For me users will use custom styles via aria-required

**Comment by anevins12** [comment]
> I don't see in which case the styles are replaced by users who could knowingly delete the *. For me users will use custom styles via aria-required

Sounds reasonable

**Comment by mbgower** [comment]
We are in the process of assessing the removal of F87 via PR #2800.
I have marked this issue to be closed if that is incorporated. If you are concerned with that approach, please comment.
